IPython/html/static/edit/less/edit.less

```diff
@@ -1,3 +1,10 @@
+.header_edit {
+}
+
+.site_edit {
+    height: 88.5%;
```

Comment:
I'm not sure I understand why this can be a problem. If the height of the window (you mean, the browser window, right?) changes, the percentage also changes accordingly.  Do you have a particular example in which this fails? This is the edit page (not the notebook), by the way.